tools/bpf/bpftool/feature.c

```diff
@@ -57,6 +58,57 @@ static void uppercase(char *str, size_t len)
 		str[i] = toupper(str[i]);
 }
 
+/* Filtering utility functions */
+
+static bool
+check_filters(const char *name, regex_t *filter_in, regex_t *filter_out)
+{
+	char err_buf[100];
+	int ret;
+
+	/* Do not probe if filter_in was defined and string does not match
+	 * against the pattern.
+	 */
+	if (filter_in != NULL) {
```

Comment:
I probably need to write some macro for that.

Reply:
Lines 72-86 repeat two almost identical blocks of code. But now I have a better solution in mind how to change that.